Resources/ServerInfo/_NF/Guidebook/Shipyard/Liquidator.xml

```diff
@@ -102,11 +102,7 @@
   
   * - Liquidator-class service ships are equipped with a single APC unit that can be used to appraise the ship's total power consumption. One can check the ship's total power consumption against the S.U.P.E.R.P.A.C.M.A.N. generator unit Target Power output: to keep SMES, substation, and APC fully charged, S.U.P.E.R.P.A.C.M.A.N. generator unit Target Power output should exceed APC's Load. Remember to check APC Load and adjust the generator unit Target Power after connecting new power-consuming machines.
   
-  ** - Liquidator-class service ships have higher power demand than standard P.A.C.M.A.N. generator unit can provide at optimal settings, so the standard P.A.C.M.A.N. was replaced with S.U.P.E.R.P.A.C.M.A.N.: recommended Target Power value to avoid blackouts during Gravity Generator charging sequence is 19 kW. Furthermore, while the portable scrubber unit is online, changing the Target Power value to 20 kW might be advisable.
-  
-  Note: Without a working generator, fully charged battery units (SMES and substation) in unmodified Liquidator-class service ships can provide power for approximately 10 minutes before fully depleting their charge.
-  
-  Note: Additional generator units can be purchased at the Frontier Station.
+  ** - Liquidator-class service ships have higher power demand than standard P.A.C.M.A.N. generator unit can provide at optimal settings, so the standard P.A.C.M.A.N. was replaced with S.U.P.E.R.P.A.C.M.A.N.: recommended Target Power value to avoid blackouts during Gravity Generator charging sequence is 21.5 kW. Furthermore, while the portable scrubber unit is online, changing the Target Power value to 22 kW might be advisable.
```

Comment:
Portable scrubber's been removed, no?  Also, consistency with the Bulker rewrite.

```suggestion
  ** - Liquidator-class service ships have higher power demand than a standard P.A.C.M.A.N. can provide at optimal settings, and come equipped with a S.U.P.E.R.P.A.C.M.A.N. instead. Please note that the recommended Target Power value to avoid blackouts during Gravity Generator charging sequence is 22 kW.
```

Reply:
_Should_ the portable scrubber be removed?  Not sure if "nasty air" is more of an engineering concern vs. a janitorial concern, but if it comes with gib cleanup, then I dunno.

Regardless, you could stick it by the dock - something like this might work, though it's a little bit in the way:
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/fe549222-3192-482c-acc4-0e11e2085c29)

Reply:
> _Should_ the portable scrubber be removed? Not sure if "nasty air" is more of an engineering concern vs. a janitorial concern, but if it comes with gib cleanup, then I dunno.
> 
> Regardless, you could stick it by the dock - something like this might work, though it's a little bit in the way

I've never ever seen anyone use it :(

Reply:
Also if there is miasma in the air, it probably comes from rotting body of ship's captain, so no point in cleaning their shuttle: they wont pay for the services.